neon_tts_plugin_coqui/__init__.py

```diff
@@ -51,6 +54,8 @@ def _get_mem_usage(self) -> str:
         Get the current process memory usage in MiB
         Returns: str memory usage in MiB
         """
+        if not self.proc:
+            return ""
```

Comment:
Looking at where it is used `LOG.debug(f"RAM={self._get_mem_usage()} MiB")`
Maybe it's better to return `???`

Reply:
Good point. Updated